ballerina/Package.md

```diff
@@ -1,17 +1,189 @@
 ## Overview
 
 [//]: # (TODO: Add overview mentioning the purpose of the module, supported REST API versions, and other high-level details.)
+[HubSpot](https://www.hubspot.com/?hubs_content=www.hubspot.com/our-story&hubs_content-cta=null) is an inbound marketing, sales, and customer service platform offering tools for CRM, content management, email marketing, lead generation, and analytics to help businesses grow and engage customers effectively.
+
+The `hubspot.crm.engagement.meeting` package offers APIs to connect and interact with [Meetings API](https://developers.hubspot.com/docs/reference/api/crm/engagements/meetings) endpoints, specifically based on HubSpot API v3.
 
 ## Setup guide
 
 [//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the HubSpot Meetings connector, you must have access to the HubSpot API through a HubSpot developer account and a HubSpot App under it. Therefore you need to register for a developer account at HubSpot if you don't have one already.
+
+### Step 1: Create/Login to a HubSpot Developer Account
+
+If you have an account already, go to the [HubSpot developer portal](https://app.hubspot.com/)
+
+If you don't have a HubSpot Developer Account you can sign up to a free account [here](https://developers.hubspot.com/get-started)
+
+### Step 2 (Optional): Create a [Developer Test Account](https://developers.hubspot.com/beta-docs/getting-started/account-types#developer-test-accounts) under your account
+
+Within app developer accounts, you can create developer test accounts to test apps and integrations without affecting any real HubSpot data.
+
+**_These accounts are only for development and testing purposes. In production you should not use Developer Test Accounts._**
+
+1. Go to Test accounts section from the left sidebar.
+
+<img src="../docs/setup/resources/test-account.png" width="70%">
+
+2. Click on the `Create developer test account` button on the top right corner.
```

Comment:
```suggestion
2. Click on the "Create developer test account" button on the top right corner.
```

Use double quotes here. Only use the backticks to highlight the ballerina keywords.
Update all other places.